openhcl/lower_vtl_permissions_guard/src/lib.rs

```diff
@@ -4,8 +4,8 @@
 //! Implements a VtlMemoryProtection guard that can be used to temporarily allow
 //! access to pages that were previously protected.
 
+#![cfg(target_os = "linux")]
```

Comment:
Put this above the doc comment. Otherwise, the doc comment will get parsed on Windows and will break one day when someone adds a reference link.

Reply:
huh TIL